docs/cspell.json

```diff
@@ -0,0 +1,686 @@
+    "myrolebinding",
+    "myrootclustertenant",
+    "mysaml",
+    "mysecret",
+    "myser",
+    "myserver",
+    "mysqld",
+    "myteam",
+    "myteleportcluster",
+    "mytenant",
+    "mytennant",
+    "myuser",
+    "myvolume",
+    "nameid",
+    "nameidentifier",
+    "nameserver",
+    "navin",
+    "ndots",
+    "netcat",
+    "netiq",
+    "netmasks",
+    "netpolicy",
+    "netsec",
+    "newapp",
+    "nginxrestarter",
+    "nistp",
+    "nocrypt",
+    "nodegroups",
+    "nodename",
+    "nohup",
+    "nologin",
+    "nosql",
+    "nowait",
+    "nvme",
+    "obtainlicense",
+    "octocat",
+    "octocats",
+    "offboarding",
+    "oidcauthentication",
+    "oids",
+    "oktapreview",
+    "oldssh",
+    "onelogin",
+    "oneshot",
+    "onmicrosoft",
+    "operatorenabled",
+    "opsexample",
+    "organisation",
+    "oustanding",
+    "oxrq",
+    "packagekit",
+    "pagerduty",
+    "pagerdutyapikey",
+    "pagerdutyapikeyfromsecret",
+    "pastable",
+    "pasteable",
+    "pgaadauth",
+    "pidof",
+    "pkill",
+    "plugindata",
+    "postgresqlselfhosted",
+    "pprof",
+    "preconfigured",
+    "prereqs",
+    "prioritise",
+    "privkey",
+    "priya",
+    "programatically",
+    "promhttp",
+    "protobuf",
+    "proxied",
+    "proxyaddr",
+    "proxying",
+    "proxylistenermode",
+    "pseudoversion",
+    "psql",
+    "pstree",
+    "ptgott",
+    "ptrace",
+    "pwgen",
+    "quicktime",
+    "rdbms",
+    "rdns",
+    "rdsproxy",
+    "readyz",
+    "realmd",
+    "reauthentication",
+    "rediscluster",
+    "redisinsight",
+    "rediss",
+    "regexes",
+    "relogged",
+    "replaceall",
+    "replicaset",
+    "requirepass",
+    "reversetunnel",
+    "reviewee",
+    "rffx",
+    "roadmap",
+    "rolearn",
+    "rolebinding",
+    "rollouts",
+    "rootcluster",
+    "rtrzn",
+    "runcommand",
+    "runscript",
+    "russjones",
+    "sadas",
+    "samlauthentication",
+    "samltest",
+    "samltestid",
+    "sampleconnector",
+    "sandboxbd",
+    "scrollback",
+    "secretname",
+    "selectnongalleryapp",
+    "selectsaml",
+    "selfhosted",
+    "selfsubjectaccessreviews",
+    "selfsubjectrulesreviews",
+    "serviceaccount",
+    "serviceaccountname",
+    "serviceaccounts",
+    "serviceacct",
+    "serviceid",
+    "sesssion",
+    "setspn",
+    "sharded",
+    "signup",
+    "slacktokenfromsecret",
+    "snowsql",
+    "splunkd",
+    "splunkd",
+    "splunkforwarder",
+    "sqlserver",
+    "sshcacerts",
+    "sshcert",
+    "sslmode",
+    "starttls",
+    "statefulset",
+    "storageenabled",
+    "subkind",
+    "sudoer",
+    "syscalls",
+    "tadmin",
+    "tbot",
+    "tbotrole",
+    "tcpout",
+    "tctl",
+    "telenode",
+    "teleportauditlogssofailed",
+    "teleportdemo",
+    "teleporters",
+    "teleportproxy",
+    "teleportroles",
+    "teleportusers",
+    "teleportversionoverride",
+    "teleportyaml",
+    "tenantname",
+    "testuser",
+    "thisisunsafe",
+    "thred",
```

Comment:
The only instance of "thred" in the docs is "thredUP". Is there a config that maybe makes this check case insensitive? As with this addition a misspelling of "thred" for "thread" will be ignored.

Reply:
I removed this entry in bbdc3421240f55fc14cb439a081efca6c3480852 and the spell checker didn't flag any misspellings.